examples/simple/src/posts/PostCreate.tsx

```diff
@@ -116,6 +116,15 @@ const PostCreate = () => {
                     source="title"
                     validate={required('Required field')}
                 />
+                <AutocompleteInput
```

Comment:
Please don't update the simple example, add a Storybook instead

Reply:
I am working on it, as said on PR description